### ISSUE TITLE
RFE: Add arm64 image to quay.io/libpod/alpine_nginx

### ISSUE THREAD

**Issue by tnk4on** [containers/podman#22572]
### Feature request description

quay.io/libpod/alpine_nginx has only amd64 image and no arm64 image. Request to add arm64 image to quay.io/libpod/alpine_nginx.

### Suggest potential solution

`pkg/machine/e2e/basic_test.go` uses quay.io/libpod/alpine_nginx.
However, there is no arm64 image, so Apple Silicon mac runs in emulation mode.

### Additional context

```
% make localmachine FOCUS="Volume ops"
...
  Machine "196dffab52c7" started successfully
  /Users/shtanaka/dev2/podman/bin/darwin/podman -r run -v /private/tmp/ginkgo844281868:/test:Z quay.io/libpod/alpine_nginx ls /test/attr-test-file
  Trying to pull quay.io/libpod/alpine_nginx:latest...
  Getting image source signatures
  Copying blob sha256:d2c7362ca710ad35a846a34571a7c3450ea3cce04efcbcb4d3af276eda154ade
  Copying blob sha256:df9b9388f04ad6279a7410b85cedfdcb2208c0a003da7ab5613af71079148139
  Copying blob sha256:71895e83ea49901b7b752bbf3ca19a54148a5f4ab5fdff3dca9bcd59d44c59e3
  Copying config sha256:ecea49d99daa5bd62ebaef1338f6bc4c948bf2651b139160404f9c1c48fcd85c
  Writing manifest to image destination
  WARNING: image platform (linux/amd64) does not match the expected platform (linux/arm64)
  /test/attr-test-file
...
  ```

**Comment by Luap99** [comment]
I don't think it is intentional but I think it is good to have at least one test that ensures x86-64 emulation works in the machine. However this should likely be en explicit check on not something unrelated like here.

**Comment by tnk4on** [comment]
Hi @Luap99,

I investigated this issue and confirmed that **it has been resolved in Podman v5.5.0+**.

## Summary of changes

The `pkg/machine/e2e/basic_test.go` now uses `quay.io/libpod/testimage:20241011` instead of `quay.io/libpod/alpine_nginx`. The `testimage:20241011` is a **multi-arch image** supporting amd64, arm64, ppc64le, and s390x.

## Version comparison

| Version | Image used in `pkg/machine/e2e/basic_test.go` |
|---------|----------------------------------------------|
| v5.0.0 ~ v5.4.x | `quay.io/libpod/alpine_nginx` (amd64 only) ❌ |
| **v5.5.0 ~ main** | `quay.io/libpod/testimage:20241011` (multi-arch) ✅ |

## Verification on Apple Silicon Mac (arm64)

I ran the original reproduction test on the latest main branch:

```
$ make localmachine FOCUS="Volume ops"
...
Trying to pull quay.io/libpod/testimage:20241011...
Getting image source signatures
Copying blob sha256:...
...
SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 77 Skipped
```

**No platform mismatch warning** - the native arm64 image was pulled.

## Conclusion

The original problem has been resolved. Closing this issue.
